### ISSUE TITLE
Create a 'report this ad' voice command

### ISSUE THREAD

**Issue by trentrand** [trentrand/brandAR-Mobile#1]
Make the voice command invoke a `reportAd(adId)` function.

`private void reportAd(int adId) {
    // call REST api endpoint (/api/reportAd/000001)
}`

http://stackoverflow.com/documentation/c%23/1971/performing-http-requests/6456/creating-and-sending-an-http-post-request#t=201703160654523487672

**Comment by MegaLuis** [comment]
I don't know what the fuck this means lol I'll read it again in the morning
and talk to you.

**Comment by trentrand** [comment]
I want to get your guys app talking to server, so I thought an easy first step would be to try calling this server api call. Then you can move to trying to request the asset zip from the server.

Follow [this guide](https://docs.microsoft.com/en-us/aspnet/web-api/overview/advanced/calling-a-web-api-from-a-net-client) to learn about GET / POST / PUT / DELETE web calls, they're the four fundamentals web api calls. GET and DELETE are self explanatory. POST creates something new, PUT updates something.

Basically you'll create a POST request to brandAR.io/api/reportAd/000001, just [reference the guide](https://docs.microsoft.com/en-us/aspnet/web-api/overview/advanced/calling-a-web-api-from-a-net-client#sending-a-post-request-to-create-a-resource) to figure it out, or ask me for help.
